backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Services/PgSqlParser.py

```diff
@@ -0,0 +1,26 @@
+import sys
+import sqlglot
+
+filename = sys.argv[1]
+with open(filename) as data_file:
+    sql = data_file.read()
+
+def format_postgres(inputSql):
+    if(has_syntax_error(inputSql) == False):
+        return '\n'.join(result)
+    else:
+        return syntax_errors
```

Comment:
i guess there is no need for `else` here

Reply:
I tried without `else` and it seems that it is needed